src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -293,7 +293,7 @@ describe("Shipping", () => {
           Me: () => meWithoutAddress,
         })
 
-        expect(screen.getAllByRole("button", { name: "Offer" }).length).toBe(2)
+        expect(screen.getAllByRole("button", { name: "Offer" }).length).toBe(1)
```

Comment:
[Commenting on the other identical fix to start a new thread] Before merging, do we want to understand _why_ this palette upgrade caused there to no longer be two elements with `role=button`?

From discussing with the web practice, I think we are ok with deploying the fix and letting the mystery be.